src/lib/remark-plugins/remark-tfe-content-exclusion/index.ts

```diff
@@ -0,0 +1,265 @@
+import { SemVer, gt, gte, lt, lte } from 'semver'
+import * as unified from 'unified'
+
+import { visit } from 'unist-util-visit'
+import type { Literal, Position, Parent } from 'unist'
+import type { TestFunctionAnything } from 'unist-util-is'
+
+class ContentExclusionError extends Error {
+	constructor(message, position: Position) {
+		super(message)
+		this.name = 'ContentExclusionError'
+		this.message = message + `\n` + ` - at line: ${position.start.line}`
+	}
+}
+
+export interface TfeContentExclusionOptions {
+	version: string
+}
+/**
+ * A remark plugin to enable
```

Comment:
Let's finish filling this doc comment out 👀

Reply:
🤦 😁

Reply:
Done! https://github.com/hashicorp/dev-portal/pull/1196/commits/714ef05b1e725040d1fa5a3f43c404c6f6a4013b